app/helpers/filters_helper.rb

```diff
@@ -27,4 +27,12 @@ def organisations_filter_options(user)
     organisation_options = user.support? ? Organisation.all : [user.organisation] + user.organisation.managing_agents
     [OpenStruct.new(id: "", name: "Select an option")] + organisation_options.map { |org| OpenStruct.new(id: org.id, name: org.name) }
   end
+
+  def collection_year_options
+    if FeatureToggle.collection_2023_2024_year_enabled?
+      { "2023": "2023/24", "2022": "2022/23", "2021": "2021/22" }
+    else
+      { "2022": "2022/23", "2021": "2021/22" }
+    end
+  end
```

Comment:
Looks good, but I'm wondering if we can get these years from form handler

Reply:
And we can always return the current and previous form, without having to check the toggle

Reply:
Agreed. We should try and find out how to dynamically generate this list.

Reply:
So what I was thinking was something like:
`previous_year = FormHandler.instance.forms["previous_lettings"].start_date.year`
`current_year = FormHandler.instance.forms["current_lettings"].start_date.year`

And then construct the options from those.
It's for a specific type of form (lettings) but I thing our collection years should align.

We shouldn't need a filter for the next collection window, because there wouldn't be any logs yet and I'm not sure if we should be showing the logs before the previous collection year in general, because I wouldn't expect to keep a record of `previous_previous_form` etc. Hence only previous and current year filters

The point around keeping/displaying logs from 2 years ago needs some more general discussion I think

Reply:
In regards to your point about dynamically generating forms, I think the current form depends on the current time, so it will switch to 23/24 form in April automatically, leaving us with 22/23 and 23/24 forms only (and a 24/25 next form I think which will look exactly as 23/24 but we shouldn't use that anyway)